homepage/templates/faq.html

```diff
@@ -0,0 +1,67 @@
+{% extends "base.html" %}
+
+{% block title %}FAQ{% endblock title %}
+
+{% block content %}
+<section id="features" class="features section custom-reduce-padding-2">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12 col-sm-12 col-xs-12">
+                <h1 class="title">FAQ</h1>
+                <div class="content">
+                    <h2>I am having trouble installing Orange.</h2>
```

Comment:
There is no way to ref a single FAQ entry. I.e., questions are missing anchor links (or `id=` attributes).

It'd be _great_ if one could "see https://orange.biolab.si/faq.html#q-trouble-installing".

Reply:
Please, suggest how do implement this.

Reply:
Writing raw HTML manually, I see no other way than to continue like so.
```html
<h2 id="q-slug">full text</h2>
```
https://en.wikipedia.org/wiki/Semantic_URL#Slug

Reply:
Ah, sorry, the `id="q-[SLUG STRING]"` was meant as a placeholder. Should contain the (approximate) [slug](https://en.wikipedia.org/wiki/Semantic_URL#Slug) of each question. Then some of us will be able to quote specific Q&A sections like:  https://orange.biolab.si/faq.html#q-specific-question-x

Reply:
Yes, I wasn't sure about that, but the internet was silent on this topic (I tried searching html slug code, html slug id, etc.). Then I thought you'd go for <> as placeholder marker. 😆 Not very html-friendly, so yeah.
So I only need to set this id and nothing else? Can fix, no prob.